src/connect/ConnectWizard.ts

```diff
@@ -51,93 +52,34 @@ export class ConnectWizard {
             type: targetResourceType
         });
 
-        if (targetResourceType.toLowerCase() === `pod`) {
-            // We skip the isolation step when debugging pods
-            this.NumberOfSteps = 3;
+        const kubectlClient = await this._binariesUtility.tryGetKubectlAsync();
+        const bridgeClient = await this._binariesUtility.tryGetBridgeAsync();
+        let currentContext: IKubeconfigEnrichedContext = null;
+        if (kubectlClient != null) {
+            currentContext = await kubectlClient.getCurrentContextAsync();
+        }
+
+        if (kubectlClient == null || bridgeClient == null || currentContext == null) {
```

Comment:
If we move this just after where we assign `bridgeClient`, we can skip the null check on line 58.